### PR TITLE
Create kitten.json

### PR DESCRIPTION
**Pull request by kittensaredabest**

### CODE REVIEW

**Review by hackermondev — DISMISSED** [review]

File: domains/kitten.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "a cool website",
+  "repo": "https://github.com/KittensAreDaBest/is-a-dev",
```
Comment (TheBotlyNoob) [inline domains/kitten.json:3]: This should be the repo for the website.

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

File: domains/kitten.json (hunk 1 above)
Comment (KittensAreDaBest) [reply domains/kitten.json:3]: > This should be the repo for the website.

wym by that

Comment (KittensAreDaBest) [reply domains/kitten.json]: ah i see now. should be fixed

**Review by TheBotlyNoob — APPROVED** [review]

**Comment by TheBotlyNoob** [comment]
Not sure this is what the website should look like...

image

**Comment by KittensAreDaBest** [comment]
> Not sure this is what the website should look like...
> 
> image

oh forgot to do the nginx config for it

**Comment by KittensAreDaBest** [comment]
> > Not sure this is what the website should look like...
> > image
> 
> oh forgot to do the nginx config for it

should be fixed now

**Comment by TheBotlyNoob** [comment]
nope...

**Comment by KittensAreDaBest** [comment]
> nope...

working for me hmm
chrome_FqC3KgOSnQ

**Comment by TheBotlyNoob** [comment]
wait, I had cache, lol

**Review by hackermondev — APPROVED** [review]

**Review by mtgsquad — APPROVED** [review]

**Pull request merged**